main.go

```diff
@@ -903,17 +966,58 @@ func transformLink(args []string) ([]string, error) {
 	return append(flags, paths...), nil
 }
 
+func splitFlagsFromArgs(all []string) (flags, args []string) {
+	for i := 0; i < len(all); i++ {
+		arg := all[i]
+		if !strings.HasPrefix(arg, "-") {
+			return all[:i], all[i:]
+		}
+		if booleanFlags[arg] || strings.Contains(arg, "=") {
+			// Either "-bool" or "-name=value".
+			continue
+		}
+		// "-name value", so the next arg is part of this flag.
+		i++
+	}
+	return all, nil
+}
+
+var booleanFlags = map[string]bool{
```

Comment:
What about putting this decl above splitFlagsFromArgs

Reply:
:shrug: I don't have a strong feeling here, but I feel like the people who try to put definitions before uses are used to it because of old C versions and compilers. In general, Go is written the opposite way - the main function goes at the top, calling functions further below. In the same way, I find it easier to put the caller above the callee. The program reads more naturally to a human that way, and the more important code tends to be further up in the file.